src/main/java/calculator/Numbers.java

```diff
@@ -0,0 +1,49 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class Numbers {
+    private static final String DEFAULT_DELIMITER_REGEX = "[,:]";
+    private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("//(.)\n(.*)");
+    private static final int DELIMITER_GROUP_INDEX = 1;
+    private static final int NUMBERS_GROUP_INDEX = 2;
+    private static final String ZERO = "0";
+
+
+    private final List<Number> numbers;
+
+    public Numbers(String text) {
+        this.numbers = textToNumbers(text);
+    }
```

Comment:
생성자에서 List 그대로 받으면 문자열을 분리하는 책임을 다른 곳으로 위임할 수 있을 것 같은데 어떻게 생각하시나요?
```suggestion
    public Numbers(List<Number> numbers) {
        this.numbers = numbers;
    }
```